pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -957,6 +962,10 @@ providers: {}
 								Missing:  []authorizationv1.ResourceAttributes{{Resource: "clusterroles", Verb: "list"}},
 							},
 						},
+						PinnedResources: []configv1.GroupVersionResource{
+							{Group: "apps", Version: "v1", Resource: "Deployment"},
+							{Group: "core", Version: "v1", Resource: "ConfigMap"},
```

Comment:
"core" or ""? I'm fine with both, but we should align this then with the enhancement doc example.